README.md

```diff
@@ -0,0 +1,56 @@
+<p align="center">
+  <img alt="Starklings logo" width="150" src="https://raw.githubusercontent.com/onlydustxyz/marketplace-frontend-old/develop/docs/resources/img/logo.png">
+</p>
+
+# Starklings App
+
+A web interactive tutorial to start learning Cairo and Starknet.
+
+## About
+
+Starklings App is a web interactive platform designed to help users kickstart their journey into learning Cairo, with a focus on [Starklings](https://github.com/shramee/starklings-cairo1) exercises. These exercises are considered essential for those taking their first steps in understanding Cairo. 
+Our goal is to simplify the initial learning experience by eliminating the need for complex configurations and installations.
```

Comment:
Starklings App is a web interactive platform designed to help users kickstart their journey into learning Cairo. 

This platform is based on [Starklings](https://github.com/shramee/starklings-cairo1) exercises, considered essential for those taking their first steps in understanding Cairo. 

Our goal is to simplify the Starklings experience by getting rid of complex configurations and installations. Just access the app on a browser and start coding.